package-lock.json

```diff
@@ -8899,8 +8899,7 @@
                 },
```

Comment:
We should think about removing the package-lock (putting in gitignore) and only using yarn exclusively.